src/bitcoind/poller.rs

```diff
@@ -541,7 +541,11 @@ fn cpfp_package(
 }
 
 // `target_feerate` is in sats/kWU
-fn should_cpfp(bitcoind: &BitcoinD, tx: &impl CpfpableTransaction, target_feerate: u64) -> bool {
+pub fn should_cpfp(
```

Comment:
nit: please make the docstring a doc comment (`///`) if you make this function public